Session9/src/com/itschool/homework/Session9Challenge12.java

```diff
@@ -0,0 +1,30 @@
+package com.itschool.homework;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+public class Session9Challenge12 {
+    public static void main(String[] args) {
+        Session9Challenge12 run = new Session9Challenge12();
+        run.convertToTimeZone();
+        // input1 example: 2022-04-03 14:30:00
+        // input2 example: America/New_York
+        // trebuie sa mapez abrevierea EST, PST etc la fusul orar stiut de Java
+    }
+
+    public void convertToTimeZone() {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Please insert date & time in format YYYY:MM:DD HH:MM:SS");
```

Comment:
won't work in this format (throws exception). It should be `YYYY-MM-DD`, since at line 24 you are parsing by this format.